### PR TITLE
New Channels, Playbooks, and Boards User Guide Landing pages

### PR DESCRIPTION
**Pull request by BenLloydPearson**
#### Summary
* Create new landing pages for the Channels, Playbooks, and Boards User guide
* Deprecate the Welcome to Mattermost and the Accessing Your Workspace pages because their content is duplicated on other pages. I created redirect rules to these locations.
* Updated all references to /guides/messaging to /guides/channels.
* Update home page messaging for Channels, Playbooks, and Boards
* Remove the Focalboard installation instructions because I suspect it won't be relevant after the launch of 6.0 and we can rely on the Focalboard site in the meantime. Is this the case? Let me know if I need to reverse this deletion.

Some things that still need to be done:
* The Extend Mattermost section at the end of the Channels guide. There is a [separate effort](https://github.com/mattermost/mattermost-developer-documentation/pull/916) to find a new home for that content and I didn't want to disrupt that.
* Migrating the /messaging directory to a new /channels directory to match the updated messaging. I think we should handle that in a separate PR so it's not too disruptive. This is already a fairly large PR and I didn't want to complicate things further. 

#### Ticket Link
N/a

### CODE REVIEW

File: source/conf.py [hunk 1]
```diff
@@ -434,7 +434,11 @@ def setup(app):
         "https://handbook.mattermost.com/operations/research-and-development/product/development-process",
     "mobile/mobile-hpns": "https://docs.mattermost.com/deploy/mobile-hpns.html",
     "guides/install-deploy-upgrade-scale": "https://docs.mattermost.com/guides/deployment.html",
-    "guides/setup-onboard-manage-comply.html": "https://docs.mattermost.com/guides/administration.html"  
+    "guides/setup-onboard-manage-comply.html": "https://docs.mattermost.com/guides/administration.html",
+    "messaging/welcome-to-mattermost-messaging.html": "https://docs.mattermost.com/overview/index.html",
+    "messaging/accessing-your-workspace.html": "https://docs.mattermost.com/messaging/signing-in.html",
+    "guides/messaging": "https://docs.mattermost.com/guides/channels.html",
+    "/focalboard/installing-boards": "https://docs.mattermost.com/guides/boards.html"
```
Comment (cwarnermm) [inline source/conf.py:441]:
```suggestion
    "focalboard/installing-boards": "https://docs.mattermost.com/guides/boards.html"  
```

**Review by cwarnermm — APPROVED** [review]
Great updates, @BenLloydPearson! I corrected the last redirect listed in the ``conf.py`` that was preventing the build from completing and the preview from being generated.

**Comment by justinegeffen** [comment]
Thanks Ben! 

Just FYI there is a PR with changes to the Boards content that I'm not quite finished and which may impact this work as there's some new content. Let's go ahead with this though, as I don't want my PR to be a blocker for this. However, the Focalboard/installing-boards content needs to stay in as it deals with installing the app which AFAIK is still being used for/post 6.0. @chenilim can clarify that though. I imagine an entry for it would be something like "Install the standalone boards app, Focalboard, on your desktop" with a link to that page. 

I'm still reviewing, but wanted to address those two points upfront. :)

**Comment by chenilim** [comment]
Thanks @BenLloydPearson and @justinegeffen! I agree that the Focalboard (Personal Server + Personal Desktop apps) instructions should be removed or greatly de-emphasized on docs.mattermost.com. Those instructions will live on at focalboard.com, which we can link to from the overview or a footnote (if at all). It's important that the main call-to-action in docs is to install Mattermost, which includes Boards.

For reference, the focalboard.com site is built and (manually) deployed from [the source in the repo here](https://github.com/mattermost/focalboard/tree/main/website).

**Comment by justinegeffen** [comment]
Thanks @chenilim! Ok that works for me then - and makes perfect sense. Out of interest, what is the plan for maintaining the Focalboard app docs going forward?

**Comment by chenilim** [comment]
Good question Justine. I think the default answer is to leave them alone for now. There will be a few small changes to rename the "Boards plugin" reference, and link to docs.mattermost.com.

The only main content overlap might be in the [User's Guide](https://www.focalboard.com/guide/user/), which can be downstream of updates to docs.mattermost.com - i.e. we can periodically review and copy/paste relevant new sections back. We also occasionally get community contributions to the docs (not many so far though).

Longer term, we could invest in making focalboard.com more developer-focused (vs. admin focused), similar to [developers.mattermost.com](https://developers.mattermost.com/). Definitely welcome thoughts and more discussion on this.

File: source/guides/playbooks.rst [hunk 2]
```diff
@@ -1,37 +1,30 @@
...
+   
+The Basics
+----------
 .. toctree::
-   :maxdepth: 2
-
-   /playbooks/refining-and-improving.rst
+   :hidden:
+
+   Overview </boards/overview>
```
Comment (justinegeffen) [inline source/guides/playbooks.rst:16]: These are all the Boards links, just FYI. :)

**Review by justinegeffen — COMMENTED** [review]
Just one comment regarding the Playbooks landing page.

**Comment by justinegeffen** [comment]
Thanks @chenilim! 

I think separating Focalboard from Boards works perfectly. I like the idea of maintaining the docs on the Focalboard site, downstream of the user guide. My only concern is that we'd get contributions there which may be missed and then not end up in the product docs. It's also harder to maintain two sets of docs. I propose directing user guide traffic to the product docs if possible (including contributions). I can update the README and relevant pages with instructions for how to contribute documentation.

Also happy to discuss this in channel!

**Comment by BenLloydPearson** [comment]
@cwarnermm We're good to publish this. Would it be possible to merge it today or tomorrow? It's a big change so I want to make sure it's not going to be disruptive.

**Pull request merged**